phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -141,7 +137,8 @@ public void setOnRetryButtonClicked(EventHandler<? super MouseEvent> event) {
     private void showAppDetails(ApplicationDTO application, SettingsManager settingsManager) {
         final AppPanel appPanel = new AppPanel(application, themeManager, settingsManager);
         appPanel.setOnScriptInstall(this::installScript);
-        showRightView(appPanel);
+        appPanel.setMaxWidth(400);
```

Comment:
Have you tried this with the "Select, if you want to view the source repository of the scripts" setting in the User Interface Settings?
I'm not sure if everything is fully visible then (see my screenshots in #738)